ansible/roles/users/tasks/remove_users.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Get running processes
+  shell: "ps aux | grep -v grep | grep -Ew \"{{ dev_users.absent|join('|') }}\" | awk '{print $2}'"
```

Comment:
You might want `grep -Ew \"^{{ dev_users.absent|join('$|^') }}$\"` in case we ever have a dev whose username is a substring of another dev's username.

Reply:
I don't think I can do that since we're not matching whole lines

Reply:
nvm. `grep -w` already only matches whole words